packages/ignite/src/cli/cli.js

```diff
@@ -24,10 +22,20 @@ module.exports = async function run (argv) {
     .token('extensionName', 'contextExtension')
     .createRuntime()
 
+  // parse the commandLine line
+  const commandLine = minimist(argv.slice(2))
+
+  // are we asking for --version|-version|-v and nothing else?
+  const showVersionNumber = isEmpty(commandLine._) && commandLine.version || commandLine.v
```

Comment:
The latter.  Expressions evaluation goes left to right.  I think it's the `isEmpty()` parenthesis that's making this distracting.  I just checked in a simpler version which breaks it down a bit. Hope that's easier to read.